CHANGELOG.md

```diff
@@ -5,7 +5,12 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
- - Current development changes [ to be moved to release ]
+### Added
```

Comment:
Ah, can you add a `## [2.0.0] - 2025-05-XX` header above this line please? The changelog is currently saying that the features in the 2.0 release are unreleased.